main.py

```
@@ -124,249 +117,159 @@ def __init__(self, api_url: str, authorization_key: str):
+                parquet_table = pq.read_table(buffer)
                 self._data = parquet_table.to_pandas()
-
-        except requests.exceptions.RequestException as req_exc:
-            print(f"Error in request: {req_exc}")
-
-        except pyarrow.ArrowInvalid as arrow_exc:
-            print(f"Error in Arrow processing: {arrow_exc}")
-
-        except Exception as e:
-            print(f"Unexpected error: {e}")
+        except (pyarrow.ArrowInvalid, Exception) as e:
```

Ruff

Undefined name `pyarrow`

(F821)